pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -469,3 +470,130 @@ def cancel(self, job_id: str) -> dict:
         jr = res.json()
 
         return jr  # type: ignore
+
+
+class QuantinuumAPIOffline:
```

Comment:
Currently the `_api_handler` argument to the backend constructor is underscored. Now that we have a (public, documented) alternative, we should probably remove the underscore -- otherwise no-one will know how to use this.

Reply:
Will do, I think the old api handler need some documentation update, too. Will add that as well.